src/options.js

```diff
@@ -13,7 +13,7 @@ export default {
   storageType: 'localStorage',
   storageNamespace: 'vue-authenticate',
   cookieStorage: {
-    domain: window.location.hostname,
+    domain: typeof(window) === 'undefined' ? '' : window.location.hostname,
```

Comment:
Please use `isUndefined` or `isDefined` helper inside `utils.js` and extract this into a helper function like `getCookieDomainUrl()`.